### PR TITLE
Adding Cyclic Sort Algo In Java

### PR DESCRIPTION
**Pull request by VishalS-HK**
Cylic Sort is a sorting algorithm ideally used if the question
reads "given range of numbers from 0 to N or 1 to N" 

Approach used : 
Check if the number is in the correct index, if not swap, and then continue iterating.

### CODE REVIEW

**Review by Kumar-laxmi — CHANGES_REQUESTED** [review]
Kindly give some description and your approach in the issue as well as PR

**Comment by VishalS-HK** [comment]
Cylic Sort is a sorting algorithm ideally used if the question
reads "given range of numbers from 0 to N or 1 to N"

Approach used :
Check if the number is in the correct index, if not swap, and then continue iterating.

**Comment by VishalS-HK** [comment]
Cylic Sort is a sorting algorithm ideally used if the question
reads "given range of numbers from 0 to N or 1 to N"

Approach used :
Check if the number is in the correct index, if not swap, and then continue iterating.

**Comment by VishalS-HK** [comment]
Is that fine? @Kumar-laxmi

**Comment by VishalS-HK** [comment]
Or should I create a new pr?

**Review by Kumar-laxmi — CHANGES_REQUESTED** [review]
Update the repository README.md

**Comment by VishalS-HK** [comment]
Done could you check

File: README.md [hunk 1]
```diff
@@ -50,6 +50,7 @@
 | 13.  | Selection Sort | [C](https://github.com/Kumar-laxmi/Algorithms/blob/main/C/Sorting/selection_sort.c), [C++](https://github.com/Kumar-laxmi/Algorithms/blob/main/C%2B%2B/Sorting/selection_sort.cpp), [Java](https://github.com/Kumar-laxmi/Algorithms/blob/main/Java/Sorting/selection_sort.java), [Python](https://github.com/Kumar-laxmi/Algorithms/blob/main/Python/Sorting/selection_sort.py) | Sorting |
 | 14.  | AVL Tree | [C++](https://github.com/Kumar-laxmi/Algorithms/blob/main/C%2B%2B/Trees/AVLTree.cpp), [Python](https://github.com/Kumar-laxmi/Algorithms/blob/main/Python/Trees/AVLTree.py) | Trees |
 | 15.  | Decision Tree | [Python](https://github.com/Kumar-laxmi/Algorithms/blob/main/Python/Machine%20Learning/Decision%20Tree/decision_tree.py) | Machine Learning |
+| 16.  | Cyclic Sort | [Java](https://github.com/Kumar-laxmi/Algorithms/tree/main/Java/Sorting) | Sorting |
```
Comment (Kumar-laxmi) [inline README.md:53]: Place Cyclic Sort between Bubble and Heap Sort

**Review by Kumar-laxmi — CHANGES_REQUESTED** [review]

**Comment by Kumar-laxmi** [comment]
@VishalS-HK Why did you close this PR??

**Pull request closed without merging**